From c11a25f2a105bc81322fa605d947bc53ea9814de Mon Sep 17 00:00:00 2001
From: Ben Pfaff <blp@ovn.org>
Date: Mon, 23 Sep 2019 12:38:56 -0700
Subject: Documentation: Fix security mailing list address.

We don't own ovs.org, and I doubt Ojai Valley School would enjoy
receiving our email.

Reviewed-by: Greg Rose <gvrose8192@gmail.com>
Signed-off-by: Ben Pfaff <blp@ovn.org>
---
 Documentation/internals/mailing-lists.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/internals/mailing-lists.rst b/Documentation/internals/mailing-lists.rst
index 33f20277b..e8b344094 100644
--- a/Documentation/internals/mailing-lists.rst
+++ b/Documentation/internals/mailing-lists.rst
@@ -93,4 +93,4 @@ security
 The `security`__ mailing list is for submitting security vulnerabilities to the
 security team.
 
-__ security@ovs.org
+__ security@openvswitch.org
-- 
2.14.1


